networking/multiple_networks/primary_networks/about-user-defined-networks.adoc

```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-user-defined-networks"]
+= About user-defined networks
+include::_attributes/common-attributes.adoc[]
+:context: about-user-defined-networks
+
+toc::[]
+
+:featurename: `UserDefinedNetwork`
+include::snippets/technology-preview.adoc[]
+
+Before the implementation of user-defined networks (UDN), the OVN-Kubernetes CNI plugin only supported a Layer 3 topology on the primary, or main, network that all pods are attached to. This allowed for network models where all pods in the cluster were part of the same global Layer 3 network, but restricted the ability to customize primary network configurations.
+
+With user-defined networks, {product-title} cluster administrators and users can create highly customizable network configurations tailored to the needs of their applications. Compared to network attachment definitions (NADs), UDNs offer enhanced network isolation, better network flexibility, simplified network management, and other advanced capabilities such as consistent and selectable IP addresses for workloads. UDNs also allow administrators to define new Layer 2 or Layer 3 configurations on the primary network, which can be applied across namespaces or workloads. Because of this, UDNs are ideal for multi-tenancy, global network isolation, or building alternative network topologies. 
+
+The following sections further emphasize the benefits and limitations of user-defined networks, the best practices when creating a `UserDefinedNetworks` custom resource, how to create the custom resource, and additional configuration details that might be relevant to your deployment.
```

Comment:
We might want to add to this for CUDN as we update docs for 4.18 more. Something like the differences between the UDN CR and the CUDN CR. Or, how Administrators can use cluster scoped CUDN CR to create native network segmentation without the need for complex network policies. Obviously this will need tweaked, but hopefully that helps a little.